Makefile

```diff
@@ -17,21 +18,35 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
                --enable-parser=h264 \
                --enable-parser=hevc
 
-CGO_LDFLAGS := -L$(FFMPEG_BUILD)/lib
+export CGO_LDFLAGS := -L$(FFMPEG_BUILD)/lib
 ifeq ($(UNAME_S),Linux)
-	CGO_LDFLAGS := "$(CGO_LDFLAGS) -l:libjpeg.a"
+	export CGO_LDFLAGS := "$(CGO_LDFLAGS) -l:libjpeg.a"
 endif
+export PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig
 
-.PHONY: build-ffmpeg lint update-rdk module clean clean-all
+.PHONY: build-ffmpeg tool-install gofmt lint update-rdk module clean clean-all
 
 $(BIN_OUTPUT_PATH)/viamrtsp: build-ffmpeg *.go cmd/module/*.go
-	PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig \
-		CGO_LDFLAGS=$(CGO_LDFLAGS) \
 		go build -o $(BIN_OUTPUT_PATH)/viamrtsp cmd/module/cmd.go
 
-lint:
+$(BIN_OUTPUT_PATH)/viamrtsp: build-ffmpeg *.go cmd/module/*.go
+	go build -o $(BIN_OUTPUT_PATH)/viamrtsp cmd/module/cmd.go
```

Comment:
This is what the makefile looks like after this diff:
```
$(BIN_OUTPUT_PATH)/viamrtsp: build-ffmpeg *.go cmd/module/*.go
		go build -o $(BIN_OUTPUT_PATH)/viamrtsp cmd/module/cmd.go

$(BIN_OUTPUT_PATH)/viamrtsp: build-ffmpeg *.go cmd/module/*.go
	go build -o $(BIN_OUTPUT_PATH)/viamrtsp cmd/module/cmd.go
```
which duplicates a make target. Please remove the duplication

Reply:
Removed. Thanks for the catch